### PR TITLE
Happy birthday ODM

### PR DESCRIPTION
**Pull request by malarzm**
We go stable!

### CODE REVIEW

**Comment by BernardoSilva** [comment]
Nice work to get here! :+1: 
Congratulations.

image

**Comment by smolinari** [comment]
Yippeee! Way to go gents! :+1: 

Scott

File: docs/en/conf.py [hunk 1]
```diff
@@ -47,7 +47,7 @@
 # The short X.Y version.
 version = '1.0'
 # The full version, including alpha/beta/rc tags.
-release = '1.0.0-BETA13'
+release = '1.0.0'
```
Comment (jmikola) [inline docs/en/conf.py]: Thanks. I always forget about this :D

File: docs/en/reference/introduction.rst [hunk 2]
```diff
@@ -324,7 +324,7 @@ its dependencies. The easiest way to do this is with `Composer`_:
 
 ::
 
-    $ composer require "doctrine/mongodb-odm=~1.0.0-BETA13@dev"
+    $ composer require "doctrine/mongodb-odm=~1.0"
```
Comment (jmikola) [inline docs/en/reference/introduction.rst]: Should we use `^1.0`?

Comment (malarzm) [inline docs/en/reference/introduction.rst]: Doesn't really matter, `^1.0` is best with somehow like `^1.0.2` I believe

Comment (jmikola) [inline docs/en/reference/introduction.rst]: Yes, good point.

**Comment by jmikola** [comment]
LGTM.

**Comment by malarzm** [comment]
http://www.doctrine-project.org/2015/08/18/doctrine-mongodb-odm-release-1-0-0.html

**Comment by jmikola** [comment]
Three cheers for our MVP of ODM stability, @malarzm!

**Comment by jwage** [comment]
:+1: Congrats!

**Comment by gruberro** [comment]
Congrats!!

party hard :-)

**Comment by smolinari** [comment]
image

Way to go!

Scott

**Pull request merged**